neurons/validator.py

```diff
@@ -983,6 +985,28 @@ async def run(self):
                 self.comms.update_peers_with_buckets()
                 self.eval_peers = self.comms.eval_peers
 
+            if self.metagraph.hotkeys != self.hotkeys:
```

Comment:
I am not sure this metagraph would ever be updated? @distributedstatemachine correct me if I am wrong but we need to sync the metagraph state in order for this to update? I think this could be a method in the `src/tplr/chain.py` instead maybe? In the chain the metagraph is updated.